devtools-integration-tests/src/test/java/org/ballerina/devtools/cmd/TestUtils.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerina.devtools.cmd;
+
+import net.lingala.zip4j.ZipFile;
+import net.lingala.zip4j.exception.ZipException;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.testng.Assert;
+
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class TestUtils {
+    private TestUtils() {
+    }
+
+    static final String BALLERINA_HOME_DIR = "BALLERINA_HOME_DIR";
+    static final String BALLERINA_DEV_CENTRAL = "BALLERINA_DEV_CENTRAL";
+    public static final Path DISTRIBUTIONS_DIR = Paths.get(System.getProperty("distributions.dir"));
+
+    private static final PrintStream OUT = System.out;
+    private static final Path TARGET_DIR = Paths.get(System.getProperty("target.dir"));
+    private static final Path TEST_DISTRIBUTION_PATH = TARGET_DIR.resolve("test-distribution");
+    public static final Path MAVEN_VERSION = Paths.get(System.getProperty("maven.version"));
+
+    /**
+     * Execute ballerina command.
+     *
+     * @param distributionName The name of the distribution.
+     * @param sourceDirectory  The directory where the sources files are location.
+     * @param args             The arguments to be passed to the build command.
+     * @return inputream with log outputs
+     * @throws IOException          Error executing build command.
+     * @throws InterruptedException Interrupted error executing build command.
+     */
+    public static Process executeCommand(String command, String distributionName, Path sourceDirectory,
```

Comment:
```suggestion
    public static void executeCommand(String command, String distributionName, Path sourceDirectory,
```

Reply:
As the returned process is not used, we could make it `void`. Or else we can return the `process.getInputStream()`, then we could assert the output.